blind_75/encode_and_decode_strings.py

```diff
@@ -0,0 +1,112 @@
+# Design an algorithm to encode a list of strings to a string. The encoded string is then sent over the network and is decoded back to the original list of strings.
+
+# Machine 1 (sender) has the function:
+
+# string encode(vector<string> strs) {
+#   // ... your code
+#   return encoded_string;
+# }
+# Machine 2 (receiver) has the function:
+# vector<string> decode(string s) {
+#   //... your code
+#   return strs;
+# }
+# So Machine 1 does:
+
+# string encoded_string = encode(strs);
+# and Machine 2 does:
+
+# vector<string> strs2 = decode(encoded_string);
+# strs2 in Machine 2 should be the same as strs in Machine 1.
+
+# Implement the encode and decode methods.
+
+# You are not allowed to solve the problem using any serialize methods (such as eval).
+
+# Input: dummy_input = ["Hello","World"]
+# Output: ["Hello","World"]
+# ["Hello world", "hi"]
+# "Hello word#Hi"
+# Explanation:
+# Machine 1:
+# Codec encoder = new Codec();
+# String msg = encoder.encode(strs);
+# Machine 1 ---msg---> Machine 2
+
+# Input: dummy_input = [""]
+# Output: [""]
+
+"""
+-------- High Level Idea/Algorithm 
```

Comment:
@marktai

Reply:
LGTM! Good test cases, though I would have added `#20` just for completion sake (I'm convinced that your code handles `#20` as word)